src/main/java/com/puppycrawl/tools/checkstyle/checks/metrics/ClassFanOutComplexityCheck.java

```diff
@@ -79,14 +79,192 @@
  * &lt;module name="ClassFanOutComplexity"/&gt;
  * </pre>
  * <p>
- * To configure the check with a threshold of 10:
+ * Code examples:
+ * </p>
+ * <p>
+ * The check passes without violations in the following.
+ * </p>
+ * <pre>
+ * class InputClassComplexity {
+ *   Set set = new HashSet(); // Ignored due to default excludedClasses property
+ *   Map map = new HashMap(); // Ignored due to default excludedClasses property
+ *   Date date = new Date(); // Counted, 1
+ *   Time time = new Time(); // Counted, 2
+ *   Place place = new Place(); // Counted, 3
+ * }
+ * </pre>
+ * <p>
+ * The check results in a violation in the following.
+ * </p>
+ * <pre>
+ * class InputClassComplexity {
+ *   Set set = new HashSet(); // Ignored due to default excludedClasses property
+ *   Map map = new HashMap(); // Ignored due to default excludedClasses property
```

Comment:
it should be mentioned that both Map and HashMap are ignored.
Same for Set and HashSet

Reply:
Done